### PR TITLE
Treant Mirana Luna

### PR DESCRIPTION
**Pull request by Evilgrill**
lycan
----
-add reduce healing
-add crit chance bonus on agility no idea how to check that

treant 
---
-ok 

brood
----
-add 1 skill 

ursa
---
-add 2 skills

mirana
-------
ok 

luna
----
ok

fix typos in abilities.txt

i learnt that ignore aggro dont need entindex. Put it in = broke it.

### CODE REVIEW

File: game/scripts/npc/creeps/zone1/boss/brood/abilities.txt [hunk 1]
```diff
@@ -106,12 +108,43 @@
...
+			"010"
+			{
+				"var_type"					"FIELD_FLOAT"
+				"max_stacks"				"66"
+			}
+			"011"
+			{
+				"var_type"					"FIELD_FLOAT"
+				"stack_duration"				    "30"
+			}
```
Comment (fdsatrew3) [inline game/scripts/npc/creeps/zone1/boss/brood/abilities.txt:140]: 010, 011 <:

File: game/scripts/npc/creeps/zone1/boss/treant/units.txt [hunk 2]
```diff
@@ -8,7 +8,7 @@
 		"Model"						"models/items/furion/treant/abyssal_prophet_abyssal_prophet_treant/abyssal_prophet_abyssal_prophet_treant.vmdl"	// Model.
 		"BaseClass"					"npc_dota_creature"
 		"SoundSet"					"Hero_Furion.TreantSpawn"
-		"Level"						"0"
+		"Level"						"5"
```
Comment (fdsatrew3) [inline game/scripts/npc/creeps/zone1/boss/treant/units.txt:11]: 5 -> 0

File: game/scripts/vscripts/creeps/zone1/boss/lycan.lua [hunk 3]
```diff
@@ -202,7 +202,7 @@ function modifier_lycan_wound_debuff:OnIntervalThink()
     damageTable.caster = self.caster
     damageTable.target = self.target
     damageTable.ability = self.ability
-    damageTable.damage = self.dot * self.target:GetMaxHealth()
+    damageTable.damage = self.dot * self.target:GetMaxHealth() *0.03 --nerf
```
Comment (fdsatrew3) [inline game/scripts/vscripts/creeps/zone1/boss/lycan.lua:205]: wtf

File: game/scripts/vscripts/creeps/zone1/boss/lycan.lua [hunk 4]
```diff
@@ -134,7 +134,7 @@ function lycan_wound:ApplyDamageAndDebuff(target, caster)
     damageTable.caster = caster
     damageTable.target = target
     damageTable.ability = self
-    damageTable.damage = damage
+    damageTable.damage = damage*0.03 --nerf
```
Comment (fdsatrew3) [inline game/scripts/vscripts/creeps/zone1/boss/lycan.lua:137]: that too

File: game/scripts/vscripts/creeps/zone1/boss/lycan.lua [hunk 5]
```diff
@@ -947,7 +950,7 @@ function modifier_lycan_bleeding_dot:OnIntervalThink()
     damageTable.caster = self.caster
     damageTable.target = self.target
     damageTable.ability = self.ability
-    damageTable.damage = damage
+    damageTable.damage = damage * 0.03 --nerf
```
Comment (fdsatrew3) [inline game/scripts/vscripts/creeps/zone1/boss/lycan.lua:953]: ?

File: game/scripts/npc/creeps/zone1/boss/brood/boss.txt [hunk 6]
```diff
@@ -7,7 +7,7 @@
         "BaseClass"         "npc_dota_creature"  // Class of entity of link to.
         "Model"           "models/heroes/broodmother/broodmother.vmdl"  // Model.
         "SoundSet"          "Creep_Good_Range"          // Name of sound set.
-        "Level"           "0"
+        "Level"           "30"
```
Comment (fdsatrew3) [inline game/scripts/npc/creeps/zone1/boss/brood/boss.txt:10]: sad kek

Comment (Evilgrill) [reply game/scripts/npc/creeps/zone1/boss/brood/boss.txt]: fixed

**Pull request merged**